src/main/java/com/spotinst/sdkjava/model/bl/ocean/ecs/ClusterOptimizeImageConfiguration.java

```diff
@@ -0,0 +1,103 @@
+package com.spotinst.sdkjava.model.bl.ocean.ecs;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class ClusterOptimizeImageConfiguration {
+
+    private Set<String>  isSet;
+    private Boolean      shouldOptimizeEcsAmi;
+    private String       performAt;
```

Comment:
define as enumeration since it can have only defined values.

Reply:
This file is shared by @chaplabansi and it not required for launch specification. Leaving it as it is. @chaplabansi  will update this class.